README.md

```diff
@@ -11,5 +11,29 @@ _bioRxiv_ 2020 doi:10.1101/2020.11.09.355982
 - [`condor`](condor): scripts for running the analyses in a HTCondor cluster
 - [`data`](data): source and target gene lists for the network flow analysis
 
-## TODO
-- update `requirements.txt`
+## Install python dependencies
+```
+# setup virtual environment for flopro dependencies
+python -m venv env
+source env/bin/activate
+
+# install dependencies
```

Comment:
It looks like there are non-Python dependencies for visualization. We should name those in the readme even if we don't attempt to install them.